cartaya/form/resources/css/media-queries.css

```diff
@@ -0,0 +1,29 @@
+@media (max-width: 480px) {
```

Comment:
Don't use media-queries unnecessary. You should use only one media-queries for each type of screen